sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -242,6 +242,10 @@ private[hive] class SparkExecuteStatementOperation(
       val executionHiveClassLoader = sqlContext.sharedState.jarClassLoader
       Thread.currentThread().setContextClassLoader(executionHiveClassLoader)
```

Comment:
Even in sync mode, we don't use the classloader of the current thread?

Reply:
> Even in sync mode, we don't use the classloader of the current thread?

In code I mentioned in description, it will use SessionState's HiveConf's ClassLoader first